ui/lesson/TerminalChallenge.tsx

```diff
@@ -3,7 +3,7 @@
 import { useState, useEffect } from 'react'
 import { LessonDirection } from 'types'
 import { Lesson, LessonTabs, LessonTerminal } from 'ui'
-import { useMediaQuery } from 'react-responsive'
+import { useMediaQuery } from 'hooks'
```

Comment:
changing it back to `react-responsive` is fixing the error

Reply:
Is there something wrong with our useMediaQuerey fn? also is there a particular reason we have a media query hook and are not using the imported one instead?